engine/gui/src/gui.cpp

```diff
@@ -532,6 +532,31 @@ namespace dmGui
         return true;
     }
 
+    static Result MakeDynamicTextureData(DynamicTexture* dynamic_texture, uint32_t width, uint32_t height, dmImage::Type type, bool flip, const void* buffer, uint32_t buffer_size)
```

Comment:
All the dynamic texture functions did the same thing, so I just merged the code paths.